src/database/mongo.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const config = require('../../config.json');
+
+//mongodb database: "splashes"
+const uri = "mongodb+srv://brilam8:" + config.mongoPass + "@splash-pages-eqhd7.mongodb.net/splashes?retryWrites=true&w=majority";
```

Comment:
Should this he set in a config file or .env? @garrettluu

Reply:
Usually it would be in a .env so you can configure it easily in production when you deploy